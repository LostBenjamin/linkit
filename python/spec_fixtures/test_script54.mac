
$times 5
$delay 10

[start]
  (doit) <times>
  (doit) <times> <delay>

[doit]
  {{{
    red
  }}}

