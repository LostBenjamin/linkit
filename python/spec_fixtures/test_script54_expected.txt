4:set:red:flu
5:set:4,5:run:4,5,10:run
